build/storage/tests/python_system_tools/config.py

```diff
@@ -0,0 +1,43 @@
+from abc import ABC, abstractmethod
```

Comment:
On Rafal patch this `python system_tools` is located in `it/ptf_tests` dir

Reply:
done